apps/plugin-browser/src/pages/popup/popup-contents/action-center/switch-with-dark-mode.tsx

```diff
@@ -0,0 +1,23 @@
+import { Switch, switchClasses, SwitchProps } from "@mui/material";
+
+export const SwitchWithDarkMode = ({ sx, ...props }: SwitchProps) => {
+  return (
+    <Switch
+      {...props}
+      sx={[
```

Comment:
Why are we not making changes like this in the theme itself? Do we not for-see this being re-usable?

Reply:
Basically because I don't want to spend time translating it to the theme yet when it's only used here (I'm slight apprehensive about how much overriding stuff is going on in the theme and breaking something else).

I agree we should move it in there when we need dark mode elsewhere